src/GraphQL/Resolvers/ExpressionFieldResolver.cs

```diff
@@ -5,6 +5,8 @@ namespace GraphQL.Resolvers
 {
     /// <summary>
     /// Returns a value from the field's graph type's source object, based on a predefined expression.
+    /// Note: this class uses dynamic compilation and therefore allocates a relatively large amount of
+    /// memory in managed heap, ~1KB. Do not use this class in cases with limited memory requirements.
```

Comment:
I will also need to write similar comments to the methods that create this resolver.